applications/FSIApplication/python_scripts/partitioned_fsi_base_solver.py

```diff
@@ -63,15 +63,57 @@ def __init__(self, model, project_parameters):
         self.fluid_interface_submodelpart_name = coupling_settings["fluid_interfaces_list"][0].GetString()
         self.structure_interface_submodelpart_name = coupling_settings["structure_interfaces_list"][0].GetString()
 
-        # Construct the structure solver
+        ## Construct the structure solver
         self.structure_solver = python_solvers_wrapper_structural.CreateSolverByParameters(self.model, self.settings["structure_solver_settings"], self.parallel_type)
         self._PrintInfoOnRankZero("::[PartitionedFSIBaseSolver]::", "Structure solver construction finished.")
 
-        # Construct the fluid solver
+        ## Construct the fluid solver
         self.fluid_solver = python_solvers_wrapper_fluid.CreateSolverByParameters(self.model, self.settings["fluid_solver_settings"], self.parallel_type)
         self._PrintInfoOnRankZero("::[PartitionedFSIBaseSolver]::", "Fluid solver construction finished.")
 
-        # Construct the ALE mesh solver
+        ## Check the ALE settings before the mesh solver construction
```

Comment:
@rubenzorrilla why don't you use the `navier_stokes_ale_fluid_solver`? The next lines are copy-paste
I think it would make sense for you, we spent quite some time trying to figure out how it works well

Reply:
Back to the discussion again :laughing: ... Mainly because in that way I can avoid to solve the mesh problem each time the `SolveSolutionStep` of the fluid is called in problems that are "not too coupled".

Reply:
:)

But wouldn't it make more sense to add a function `SolveSolutionStepMoveMeshOnly` (bad name but you get the idea) which calls `MoveMesh` instead of `SolveSolutionStep` on the mesh-solver?

Reply:
Agree, at this point it's worth to discuss about it. However, let's do it in another PR/issue.